### ISSUE TITLE
Radius of Curvature

### ISSUE THREAD

**Issue by ThomasMGeo** [zsylvester/meanderpy#7]
Is it possible to have Radius of Curvature stats be automatically calculated?

**Comment by zsylvester** [comment]
I guess you could make centerline curvature an attribute of each Channel instance, they are computed anyway. Then one could generate whichever stats they were interested in, without having to recompute curvature. Is this similar to what you were thinking?

**Comment by ThomasMGeo** [comment]
Yeah! Just curious if it was already calc'd, without having to reinvent the wheel.
